lte/gateway/c/core/oai/lib/openflow/controller/GTPApplication.cpp

```diff
@@ -291,7 +291,7 @@ static void add_downlink_match(of13::FlowMod& downlink_fm,
 
 static void mask_ipv6_address(uint8_t* dst, const uint8_t* src,
                               const uint8_t* mask) {
-  for (int i = 0; i < INET6_ADDRSTRLEN; i++) {
```

Comment:
@themarwhal , this macro is being used in other places as well, such as PagingApplication.cpp. It will be good to fix it in all places. cc: @pshelar

Reply:
good find, its dup code, needs same fix.

Reply:
Do you mean this one? https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/core/oai/lib/openflow/controller/PagingApplication.cpp?L111 

I can fix that one too in this PR if it's the same issue.

Reply:
fixed! @ssanadhya